packages/app/src/pages/actueel/veiligheidsregio/[code].tsx

```diff
@@ -160,48 +161,59 @@ const TopicalSafetyRegion: FCWithLayout<typeof getStaticProps> = (props) => {
-                      dangerouslySetInnerHTML={{
-                        __html: text.risiconiveaus.selecteer_toelichting,
-                      }}
-                    />
-                  }
-                  legendComponent={
-                    <EscalationMapLegenda
+            <Box>
+              <TopicalTile
+                css={css({
+                  pb: 0,
+                  mb: asResponsiveArray({ _: '2rem', md: '4rem' }),
+                  px: asResponsiveArray({ _: 3, md: 4 }),
+                })}
+              >
+                <>
+                  <TopicalChoroplethContainer
+                    title={text.risiconiveaus.selecteer_titel}
+                    description={
+                      <div
+                        dangerouslySetInnerHTML={{
+                          __html: text.risiconiveaus.selecteer_toelichting,
+                        }}
+                      />
+                    }
+                    legendComponent={
+                      <EscalationMapLegenda
+                        data={choropleth.vr}
+                        metricName="escalation_levels"
+                        metricProperty="escalation_level"
+                      />
+                    }
+                  >
+                    <SafetyRegionChoropleth
                       data={choropleth.vr}
                       metricName="escalation_levels"
                       metricProperty="escalation_level"
+                      onSelect={createSelectRegionHandler(
+                        router,
+                        'maatregelen'
+                      )}
+                      tooltipContent={escalationTooltip(
+                        createSelectRegionHandler(router, 'maatregelen')
+                      )}
                     />
-                  }
-                >
-                  <SafetyRegionChoropleth
-                    data={choropleth.vr}
-                    metricName="escalation_levels"
-                    metricProperty="escalation_level"
-                    onSelect={createSelectRegionHandler(router, 'maatregelen')}
-                    tooltipContent={escalationTooltip(
-                      createSelectRegionHandler(router, 'maatregelen')
-                    )}
-                  />
-                </TopicalChoroplethContainer>
-                <Box
-                  borderTopWidth="1px"
-                  borderTopStyle="solid"
-                  borderTopColor={colors.silver}
-                  mt={3}
-                  mx={-4}
-                >
-                  <TopicalTile css={css({ mb: 0, pb: 0 })}>
-                    <EscalationLevelExplanations />
-                  </TopicalTile>
-                </Box>
-              </>
-            </TopicalTile>
+                  </TopicalChoroplethContainer>
+                  <Box
+                    borderTopWidth="1px"
+                    borderTopStyle="solid"
+                    borderTopColor={colors.silver}
```

Comment:
```suggestion
                    borderTopColor="silver"
```

Styled system should pick up the correct color.